features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/Default/APICreateDefault.jsx

```diff
@@ -254,6 +244,16 @@ function APICreateDefault(props) {
                                 </Button>
                             </Link>
                         </Grid>
+                        <Grid item>
+                            <Button
+                                variant='contained'
+                                color='primary'
+                                disabled={!apiInputs.isFormValid || isCreating}
+                                onClick={createAPI}
+                            >
+                                Create {isCreating && <CircularProgress size={24} />}
```

Comment:
Shall we add permission checking condition to this button and other places too? For this button check how this is done in general API create button.

Reply:
We won't add this to here as the logic comes with isCreating prop

Reply:
isCreating call is not doing the required permission check. Anyway that issue needs to address separately.

Reply:
Yes. It has been addressed by the root level by disabling the "Create API" button.